### DISCUSSION TITLE
Can a translation feature be added?

### DISCUSSION THREAD

**Discussion by futuretao** [helix-editor/helix#7336] [Ideas] [answered]

**Comment by futuretao** [comment]
Google Translate

**Comment by pascalkuthe** [comment, marked as answer]
I dont think translation is in scope for core. If you have a CLI tool for translation you could just pipe text into it with `:pipe`

**Reply by kirawi** [reply]
https://github.com/crow-translate/crow-translate looks like a fairly popular option. It can be used in command line.

**Comment by ptechen** [comment]
pipe is troublesome!
